ceph.spec.in

```diff
@@ -1810,13 +1785,8 @@ if test $STATUS -eq 0; then
     /usr/bin/systemctl stop ceph.target > /dev/null 2>&1
 fi
 
-# Relabel the files
-# Use ceph-disk fix for first package install and fixfiles otherwise
-if [ "$1" = "1" ]; then
-    /usr/sbin/ceph-disk fix --selinux
-else
-    /usr/sbin/fixfiles -C ${FILE_CONTEXT}.pre restore 2> /dev/null
```

Comment:
This has a disadvantage that ceph-disk fix runs in parallel per osd which can speed things up a lot on upgrade. It was originally introduced to speed up upgrade from non-selinux enabled builds to selinux enabled builds. These builds have been here for quite a while so this is probably ok at this point.